src/05-derivatives/03-imaging.md

```diff
@@ -0,0 +1,266 @@
+# Imaging data types
+
+This section pertains to imaging data, which characteristically have spatial
+extent and resolution.
+
+## Preprocessed, coregistered and/or resampled volumes
+
+Volumetric preprocessing does not modify the number of dimensions, and so
+the specifications in [Preprocessed or cleaned data][common_preproc]
+apply.
+In addition, all processed files include the following metadata JSON fields:
+
+| **Key name**  | **Description**                                                                                 |
+| ------------- | ----------------------------------------------------------------------------------------------- |
+| SkullStripped | REQUIRED. Boolean. Whether the volume was skull stripped (non-brain voxels set to zero) or not. |
```

Comment:
Does this interact with https://github.com/poldracklab/smriprep/pull/116 ?

Reply:
Not obviously, to me.

Reply:
I don't think so. As @adelavega mentioned, that PR is to make smriprep tolerant to datasets (raw) containing skull-stripped images because of human error or because they were built before BIDS-Derivatives existed and lost access to the original images.

Reply:
(in other words, you can't expect the dataset to have the flag set because of either two reasons: i) it would be invalid or ii) very likely the file was included without knowing it was skull-stripped)

Reply:
OK ! My line of thinking was that once Derivatives is merged, in order to be BIDS compliant those datasets would have the `SkullStripped` field (since being skullstripped is a derivative, if a "minimal" one) regardless of whether the original images are still available. In which case the failure would be human error in not including the field, and the fix would be adding it.

But it seems that's not the consensus, so I'll resolve this. Thanks for the clarification !